### PR TITLE
REP-17 Replace username/password with Oauth token

### PR DESCRIPTION
**Pull request by dovekie**
@alexanderyoung PTAL

### CODE REVIEW

File: default/setup.xml [hunk 1]
```diff
@@ -5,18 +5,13 @@
...
-  <input field="clientID">
-    <label>This is the Client ID that you received from Crittercism Support</label>
+    <label>Crittercism OAuth Token</label>
     <type>text</type>
   </input>
-<text> Notes:</text> 
-<text> - To change the Credentials or Crittercism API Key later, first remove the entry from SPLUNK_HOME/etc/apps/crittercism/local/app.conf and restart Splunk.
-</text><text> - The script may take up to 10 minutes to start getting data into the Crittercism dashboards.
+<text> Notes:</text>
+    <text> - To generate your Crittercism OAuth Token, please visit https://app.crittercism.com/developers/user-settings and follow the instructions on the page.</text>
```
Comment (alexanderyoung) [inline default/setup.xml]: Let's be specific here and say 'OAuth Access Token' please

File: default/setup.xml [hunk 2]
```diff
@@ -5,18 +5,13 @@
...
+    <text> - To change the Credentials or Crittercism API Key later, first remove the entry from SPLUNK_HOME/etc/apps/crittercism/local/app.conf and restart Splunk.
```
Comment (alexanderyoung) [inline default/setup.xml]: Is there a way to allow them to paste a new token in rather than editing that line?

**Comment by alexanderyoung** [comment]
Added 2 comments.

File: default/setup.xml (hunk 2 above)
Comment (dovekie) [inline default/setup.xml]: This would be its own project.

**Comment by alexanderyoung** [comment]
LGTM

**Pull request merged**